README.md

```diff
@@ -0,0 +1,9 @@
+# Selector State Fragments
+
+Converts Web Annotation
+[Selectors and States as Fragment Identifiers](https://www.w3.org/TR/selectors-states/#frags)
+into a [Web Annotation Data Model Selector](https://www.w3.org/TR/selectors-states/#selectors).
```

Comment:
More accurately, we create a Specific Resource, right? Which can *contain* a selector, or a state.